submissions/victorAienobe/dom/sign-up.js

```diff
@@ -0,0 +1,35 @@
+let submitForm = document.getElementById('validateForm');
+let firstName = document.getElementById('firstName');
+let lastName = document.getElementById('lastName');
+let email = document.getElementById('email');
+let phoneNumber = document.getElementById('phoneNumber');
+let password = document.getElementById('password');
+let confirmPassword = document.getElementById('confirmPassword');
+
+function validateRegex(field, regex, error1, error2, event) {
+    if(field.value === "") {
+      field.nextElementSibling.innerHTML = error1;
+      event.preventDefault();
+      return false;
+    } else if (!(field.value.match(regex))) {
+        field.nextElementSibling.innerHTML = error2;
+        event.preventDefault();
+        return false;
+        }
+    else {
+      field.nextElementSibling.innerHTML = "";
+      return true;
+    }
+}
+
+function validateForm(e) {
+    validateRegex(firstName, /^[a-z]+$/i, 'Please enter your First name', 'Please enter your First name', e);
```

Comment:
There are too many repetitions here. Also, we want to ensure we write clean and reusable code. I will advise you put this `Please enter your First name` within a variable instead. Since you are using it more than ones. Same goes for others too.

Meanwhile, well done for the good job done so far.